### DISCUSSION TITLE
Is there a way to link/associate channels?

### DISCUSSION THREAD

**Discussion by mcantar** [Rem0o/FanControl.Releases#2092] [Q&A]
What I'd like to be able to do is set a dynamic association between channels.

Example 1:
(I'm currently dealing with this)
I have a water loop with a dual pump setup, with the pumps in series and so needing to be ran at roughly equal speeds. Assuming the RPM reporting is accurate, one of my pumps runs slightly slower than the other, such that I need "Pump B" set to 53% to match the same RPM of "Pump 1" at 50%.
I'd like to be able to set these against a curve, with a rule that "Pump B" should always be +3% over "Pump A."

Example 2:
Say someone has two banks of fans set to intake/exhaust respectively, and wants to ensure that they always have a positive case pressure but also want to be able to use curves. They could set a rule/association that the intake fan bank shall always run +5% (or whatever) higher than the exhaust fan bank.

Is there already a way to do this in the software that I'm just not seeing? Or should I post this over in the Ideas section?

**Comment by Rem0o** [comment]
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/6632d28b-84ae-4649-9a3a-e0d77548904d)
